cogs/events.py

```diff
@@ -6,7 +6,7 @@
 
 import config
 import discord
-from discord.ext import commands, tasks
+from discord.ext import commands, tasks, flags
```

Comment:
Why do you need flags in the events file?

Reply:
see line 60

Reply:
👌